Makefile

```diff
@@ -144,14 +144,19 @@ test-typescript: ## Test Typescript templates
 test-integration: ## Run integration tests using an available cluster.
 	go test -tags integration -timeout 30m --coverprofile=coverage.txt ./... -v
 
-test-e2e: ## Run end-to-end tests using an available cluster.
+.PHONY: func-instrumented
+
+func-instrumented: ## Func binary that is instrumented for e2e tests
+	env CGO_ENABLED=1 go build -ldflags $(LDFLAGS) -cover -o func ./cmd/func
```

Comment:
I also thought about adding `-race` but that breaks tests.
We have race condition in code but I believe it's benign.